src/main/resources/db/changelog/changes/01-create-book-table.yaml

```diff
@@ -0,0 +1,57 @@
+databaseChangeLog:
+  - changeSet:
+      id: createBookTable
+      author: anigilator479
+      changes:
+        - createTable:
+            tableName: book
+            columns:
+              - column:
+                  name: id
+                  type: BIGINT
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+              - column:
+                  name: title
+                  type: VARCHAR(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: author
+                  type: VARCHAR(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: isbn
+                  type: VARCHAR(255)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: price
+                  type: DECIMAL
+                  constraints:
+                    nullable: false
+              - column:
+                  name: is_deleted
+                  type: BIT(1)
+                  constraints:
+                    nullable: false
+              - column:
+                  name: description
+                  type: VARCHAR(255)
+                  constraints:
+                    nullable: true
+              - column:
+                  name: cover_image
+                  type: VARCHAR(255)
+                  constraints:
+                    nullable: true
+
+  - changeSet:
+      id: addUniqueConstraintIsbn
+      author: anigilator479
+      changes:
+        - addUniqueConstraint:
+            tableName: book
+            columnNames: isbn
```

Comment:
is it possible to specify this constraint in the first changeSet within column definition?